recipe/meta.yaml

```
@@ -76,19 +80,25 @@ outputs:
   - name: python-gmsh
     build:
       noarch: python
-      skip: true  # [build_platform != target_platform]
-    script: install_python.sh  # [not win]
-    script: install_python.bat  # [win]
+      skip: true  # [build_platform != target_platform or target_platform != "linux-64" or occt != "7.8.1"]
```

I would leave out the occt condition here. It'll become a pain to keep up-to-date (c.f. #105, #106)
```suggestion
      skip: true  # [build_platform != target_platform or target_platform != "linux-64"]
```